src/pages/signin/LoginForm/index.native.js

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect, useRef} from 'react';
+import PropTypes from 'prop-types';
+import BaseLoginForm from './BaseLoginForm';
+import AppStateMonitor from '../../../libs/AppStateMonitor';
+
+const propTypes = {
+    /** Function used to scroll to the top of the page */
+    scrollPageToTop: PropTypes.func,
+};
+const defaultProps = {
+    scrollPageToTop: () => {},
```

Comment:
Should this be set to `undefined`? So we can early return the listener in the first render?

Reply:
Makes sense, updated